Snap Checkout new icon

spatel-snap opened the pull request:
** Checklist **
- [ ] All icons have a corresponding entry in `db/payment_icons.yml`
- [ ] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [ ] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [ ] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [ ] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

spatel-snap commented:
Hello, Can you please review this pull request?

larouxn commented:
👋 @spatel-snap, can we adjust this new icon to match the same shape/size as Visa and other icons? We can probably use the [template](https://github.com/activemerchant/payment_icons/blob/62017ee4bd124d8dc0eea23868f5c49ee84f4aa6/CONTRIBUTING.md#template).

<img width="408" alt="Screen Shot 2022-02-25 at 15 46 22" src="https://user-images.githubusercontent.com/1557529/155667910-2f350405-9252-45e0-8109-2a3af53e248c.png">

spatel-snap commented:
@larouxn  Updated the svg image as per provided template, can you please verify it again?

larouxn commented:
Hmm, interesting it still looks the same on my end. 🤔 
<img width="1242" alt="Screen Shot 2022-02-25 at 16 57 07" src="https://user-images.githubusercontent.com/1557529/155677220-fba68492-6d9b-4133-8cce-9ee17f5cec36.png">
<img width="428" alt="Screen Shot 2022-02-25 at 16 56 40" src="https://user-images.githubusercontent.com/1557529/155677169-e8716f45-cbf2-4c64-ae4c-ed8d9ad2424e.png">

spatel-snap commented:
@larouxn WIll you be merging this PR?

larouxn commented:
> @larouxn WIll you be merging this PR?

Sorry no, the icon needs to be update to be the same shape as Visa and all the other icons. It's still not the same shape.

spatel-snap commented:
@larouxn Updated the shape same as visa

spatel-snap commented:
@larouxn can you please review this icon again?

spatel-snap commented:
@larouxn  Can you please review and approved this icon?

spatel-snap commented:
@larouxn  How many time will taken for approval this icon, because there are 110 customers are waiting our plugin, can you please do it as soon as possible?

larouxn reviewed (changes requested):
Hmm, looks _almost_ good to go but it seems it's still somehow bigger than the standard icon according to Visa.
<img width="410" alt="Screen Shot 2022-03-01 at 17 06 03" src="https://user-images.githubusercontent.com/1557529/156129542-2965b53d-9ed4-4b0a-919f-b0efc2abda4e.png">

Maybe the border is off. For example I don't see the following kind of border markup in the SVG. ([template](https://github.com/activemerchant/payment_icons/blob/fe233e6baeff072069e31aefcad8464d932e72eb/CONTRIBUTING.md#template))
```SVG
<path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/>
```
<img width="1920" alt="Screen Shot 2022-03-01 at 17 08 45" src="https://user-images.githubusercontent.com/1557529/156129890-d4117b1c-61a1-4bbd-8277-57730ab8a46d.png">
<img width="1920" alt="Screen Shot 2022-03-01 at 17 08 59" src="https://user-images.githubusercontent.com/1557529/156129909-8ab59308-b460-445c-b0b3-83e5e4224bef.png">

spatel-snap commented:
@larouxn  Updated changes as per template in svg icon, can you please check and approve it?

tauthomas01 commented:
Hello,

- The border radius is off and we see the blue background over the grey border
- Please remove unused `class` attributes as per the [guidelines](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) said.

For easier comparison with other icons, open the icon in a browser, zoom in at 200% and switch tabs like [this video demonstration](https://screenshot.click/01-28-uv4tr-mer2k.mp4)

[![alt](https://screenshot.click/01-33-794vc-a8hlx.png)](https://screenshot.click/01-33-794vc-a8hlx.png)

spatel-snap commented:
@larouxn I have applied the gray border as expected and followed the standard as per template, as removed the style tag, applied the inline style, added the <path fill="#000" <path fill="#fff" and added the unique id as per template

tauthomas01 commented on app/assets/images/payment_icons/snap_checkout.svg:
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" viewBox="0 0 38 24" aria-labelledby="pi-snap_checkout" style="enable-background:new 0 0 38 24;" xml:space="preserve"><title id="pi-snap_checkout">Snap Checkout</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2  ...
```
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" viewBox="0 0 38 24" aria-labelledby="pi-snap_checkout" style="enable-background:new 0 0 38 24;" xml:space="preserve"><title id="pi-snap_checkout">Snap Checkout</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g><g><path style="fill:#1A3A67;" d="M7.94,8.66c-0.4,0.32-0.62,0.81-0.61,1.32c-0.02,0.38,0.14,0.75,0.43,1.01 c0.43,0.34,0.95,0.56,1.49,0.66c0.19,0.05,0.38,0.09,0.58,0.12c0.58,0.12,0.89,0.31,0.89,0.57c0,0.17-0.09,0.33-0.25,0.41 c-0.2,0.1-0.43,0.15-0.66,0.14c-0.31,0.03-0.62-0.04-0.89-0.2c-0.2-0.15-0.31-0.39-0.31-0.64h-1.4v0.06 c-0.03,0.54,0.19,1.07,0.61,1.41c0.52,0.36,1.14,0.52,1.77,0.48c0.66,0.05,1.33-0.11,1.9-0.46c0.44-0.34,0.7-0.87,0.67-1.43 c0.01-0.12,0.01-0.23,0-0.35c-0.07-0.32-0.26-0.6-0.53-0.78c-0.27-0.17-0.57-0.3-0.89-0.38c-0.38-0.12-0.78-0.19-1.16-0.29 c-0.19-0.05-0.38-0.11-0.55-0.2c-0.09-0.05-0.22-0.17-0.22-0.28C8.8,9.67,8.88,9.52,9.01,9.44C9.2,9.34,9.42,9.3,9.64,9.31 c0.24-0.01,0.49,0.05,0.69,0.19l0.17,0.17c0.12,0.23,0.36,0.37,0.61,0.36h0.89c-0.02-0.51-0.27-0.99-0.66-1.32 c-0.52-0.35-1.14-0.52-1.77-0.48C8.97,8.19,8.38,8.36,7.91,8.7"/><path style="fill:#1A3A67;" d="M7.94,8.66c-0.4,0.32-0.62,0.81-0.61,1.32c-0.02,0.38,0.14,0.75,0.43,1.01 c0.43,0.34,0.95,0.56,1.49,0.66c0.19,0.05,0.38,0.09,0.58,0.12c0.58,0.12,0.89,0.31,0.89,0.57c0,0.17-0.09,0.33-0.25,0.41 c-0.2,0.1-0.43,0.15-0.66,0.14c-0.31,0.03-0.62-0.04-0.89-0.2c-0.2-0.15-0.31-0.39-0.31-0.64h-1.4v0.06 c-0.03,0.54,0.19,1.07,0.61,1.41c0.52,0.36,1.14,0.52,1.77,0.48c0.66,0.05,1.33-0.11,1.9-0.46c0.44-0.34,0.7-0.87,0.67-1.43 c0.01-0.12,0.01-0.23,0-0.35c-0.07-0.32-0.26-0.6-0.53-0.78c-0.27-0.17-0.57-0.3-0.89-0.38c-0.38-0.12-0.78-0.19-1.16-0.29 c-0.19-0.05-0.38-0.11-0.55-0.2c-0.09-0.05-0.22-0.17-0.22-0.28C8.8,9.67,8.88,9.52,9.01,9.44C9.2,9.34,9.42,9.3,9.64,9.31 c0.24-0.01,0.49,0.05,0.69,0.19l0.17,0.17c0.12,0.23,0.36,0.37,0.61,0.36h0.89c-0.02-0.51-0.27-0.99-0.66-1.32 c-0.52-0.35-1.14-0.52-1.77-0.48C8.97,8.19,8.38,8.36,7.91,8.7"/></g><g> <path style="fill:#1A3A67;" d="M12.45,10.59L12.45,10.59v3.31h1.47v-3.36c0-0.54,0.44-0.97,0.97-0.98 c0.54,0,0.97,0.44,0.98,0.97c0,0,0,0,0,0v2.66c0,0.19,0.07,0.38,0.19,0.52c0.14,0.13,0.33,0.2,0.52,0.19h0.73v-3.35 c0-1.35-1.09-2.44-2.44-2.44s-2.44,1.09-2.44,2.44"/><path style="fill:#1A3A67;" d="M12.45,10.59L12.45,10.59v3.31h1.47v-3.36c0-0.54,0.44-0.97,0.97-0.98 c0.54,0,0.97,0.44,0.98,0.97c0,0,0,0,0,0v2.66c0,0.19,0.07,0.38,0.19,0.52c0.14,0.13,0.33,0.2,0.52,0.19h0.73v-3.35 c0-1.35-1.09-2.44-2.44-2.44s-2.44,1.09-2.44,2.44"/></g><g><path style="fill:#1A3A67;" d="M19.14,12.74c-0.26-0.21-0.3-0.59-0.09-0.85c0.01-0.01,0.02-0.03,0.03-0.04 c0.23-0.27,0.73-0.29,1.08-0.36c0.3-0.05,0.6-0.13,0.89-0.25v0.48c0.02,0.32-0.11,0.63-0.35,0.83c-0.27,0.21-0.61,0.33-0.96,0.3 C19.59,12.85,19.14,12.74,19.14,12.74z M18.43,8.61c-0.34,0.26-0.55,0.65-0.59,1.07h1.4c0.06-0.15,0.19-0.27,0.35-0.32 c0.38-0.13,1.45-0.2,1.45,0.43c0,0.48-0.8,0.53-1.13,0.58c-0.6,0.04-1.19,0.23-1.7,0.56c-0.41,0.31-0.62,0.82-0.55,1.33 c-0.05,0.52,0.16,1.03,0.57,1.36c0.38,0.28,0.85,0.42,1.32,0.42c0.3,0,0.59-0.02,0.89-0.05c0.25-0.03,0.5-0.09,0.73-0.19l0,0 c0.12,0.14,0.3,0.23,0.49,0.24h0.72V9.75c0.02-0.47-0.22-0.92-0.61-1.18c-0.54-0.3-1.16-0.44-1.77-0.4 c-0.58-0.04-1.15,0.13-1.61,0.49"/></g><g><path style="fill:#1A3A67;" d="M29.81,7.04c-0.4,0-0.73,0.33-0.73,0.73l0.4,4.23h1.07l0.24-4.95L29.81,7.04z"/><path style="fill:#1A3A67;" d="M29.81,7.04c-0.4,0-0.73,0.33-0.73,0.73l0.4,4.23h1.07l0.24-4.95L29.81,7.04z"/></g><g><path style="fill:#1A3A67;" d="M29.37,12.57v1.36h0.8c0.25,0.03,0.48-0.15,0.51-0.4c0-0.04,0-0.08,0-0.11v-0.89L29.37,12.57 z"/><path style="fill:#1A3A67;" d="M29.37,12.57v1.36h0.8c0.25,0.03,0.48-0.15,0.51-0.4c0-0.04,0-0.08,0-0.11v-0.89 L29.37,12.57z"/></g><path style="fill:#98C354;" d="M7.64,16.97v-1.38H7.4v-0.29h0.24v-0.27c0-0.27,0.17-0.42,0.5-0.42H8.4v0.35H8.28c-0.15,0-0.23,0.06-0.23,0.21 v0.12h0.36v0.28H8.05v1.39H7.64z"/><path style="fill:#98C354;" d="M9.01,14.61h0.41v0.42H9.01V14.61z M9.01,15.29h0.41v1.68H9.01V15.29z"/><path style="fill:#98C354;" d="M10.19,16.97v-1.67h0.41v0.17c0.1-0.13,0.25-0.2,0.42-0.2c0.28-0.04,0.53,0.15,0.58,0.42 c0.01,0.05,0,1.3,0,1.3h-0.36v-1.02c0-0.2-0.1-0.3-0.28-0.3c-0.09,0-0.18,0.03-0.25,0.1c-0.08,0.1-0.11,0.22-0.11,0.35v0.89h-0.4 V16.97z"/><path style="fill:#98C354;" d="M13.82,16.69c-0.01,0.1,0.01,0.19,0.05,0.28h-0.42c-0.03-0.04-0.05-0.1-0.04-0.15 c-0.15,0.12-0.34,0.19-0.54,0.19c-0.15,0.01-0.29-0.04-0.41-0.14c-0.11-0.1-0.17-0.24-0.16-0.38c0-0.4,0.67-0.52,0.67-0.52 l0.43-0.16l0,0c0-0.14-0.1-0.2-0.32-0.2c-0.08-0.01-0.16,0.01-0.23,0.05c-0.05,0-0.07,0.1-0.07,0.19h-0.4 c-0.01-0.17,0.07-0.33,0.21-0.43c0.15-0.09,0.32-0.14,0.49-0.13c0.17-0.01,0.34,0.02,0.5,0.1c0.16,0.09,0.25,0.27,0.23,0.45 M13.41,16.4v-0.24c-0.06,0.03-0.12,0.05-0.19,0.06L13,16.28l-0.19,0.06c-0.05,0.04-0.09,0.1-0.08,0.17c0,0.06,0.03,0.11,0.07,0.14 c0.05,0.04,0.12,0.06,0.19,0.06c0.1,0,0.2-0.02,0.28-0.07C13.35,16.59,13.41,16.5,13.41,16.4"/><path style="fill:#98C354;" d="M14.56,16.97v-1.67h0.41v0.17c0.11-0.13,0.28-0.21,0.45-0.2c0.28-0.04,0.54,0.16,0.58,0.44 c0.01,0.04,0.01,0.09,0,0.13v1.14h-0.4v-1.02c0-0.2-0.09-0.3-0.28-0.3c-0.09,0-0.18,0.03-0.25,0.1c-0.07,0.1-0.11,0.22-0.1,0.35 v0.89h-0.41V16.97z"/><path style="fill:#98C354;" d="M18.25,15.73l-0.35,0.17c-0.03-0.08-0.07-0.15-0.13-0.2c-0.06-0.05-0.14-0.07-0.22-0.07 c-0.13,0-0.25,0.07-0.32,0.18c-0.08,0.1-0.11,0.23-0.11,0.35c0,0.12,0.03,0.24,0.1,0.35c0.07,0.1,0.19,0.16,0.31,0.16 c0.19-0.01,0.36-0.14,0.41-0.33l0.34,0.19c-0.11,0.32-0.42,0.52-0.75,0.51c-0.23,0.01-0.45-0.08-0.6-0.25 c-0.15-0.17-0.23-0.4-0.22-0.63c-0.01-0.23,0.07-0.46,0.23-0.64c0.17-0.17,0.39-0.25,0.63-0.24c0.31-0.02,0.6,0.17,0.7,0.47"/><path style="fill:#98C354;" d="M20.44,16.23h-1.23c0,0.12,0.05,0.23,0.14,0.3c0.08,0.07,0.18,0.11,0.29,0.12c0.19,0.01,0.37-0.08,0.48-0.24 l0.34,0.19c-0.17,0.28-0.41,0.42-0.82,0.42c-0.27,0-0.42-0.07-0.63-0.26c-0.15-0.17-0.23-0.4-0.22-0.63 c-0.01-0.23,0.07-0.46,0.23-0.63c0.17-0.16,0.39-0.25,0.62-0.25C20.19,15.28,20.39,15.59,20.44,16.23 M19.21,15.92h0.82 c-0.01-0.07-0.04-0.14-0.09-0.19c-0.07-0.09-0.18-0.14-0.29-0.14c-0.11,0-0.21,0.03-0.29,0.1C19.28,15.75,19.23,15.83,19.21,15.92"/><path style="fill:#1A3A67;" d="M25.68,8.17c-1.58,0-2.85,1.28-2.85,2.86l0,0v4.92h0.71c0.19,0,0.37-0.08,0.51-0.21 c0.14-0.13,0.21-0.32,0.2-0.51v-1.77c0.43,0.26,0.92,0.4,1.43,0.4c1.58,0,2.86-1.28,2.86-2.86c0-1.58-1.28-2.86-2.86-2.86 L25.68,8.17z M25.73,12.49c-0.81,0-1.47-0.65-1.47-1.45c0-0.81,0.65-1.47,1.45-1.47c0.81,0,1.47,0.65,1.47,1.45 c0,0.01,0,0.01,0,0.02C27.18,11.84,26.53,12.49,25.73,12.49z"/></g></svg>
```

tauthomas01 reviewed (changes requested):
Almost there. You just need to remove the unused `id` attributes and you're good to go.

spatel-snap commented:
@larouxn Removed the unused id attribute as you suggested. Can you please check this again?

spatel-snap commented:
@larouxn  please review this as soon as possible.

tauthomas01 commented on app/assets/images/payment_icons/snap_checkout.svg:
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" viewBox="0 0 38 24" aria-labelledby="pi-snap_checkout" style="enable-background:new 0 0 38 24;" xml:space="preserve"><title id="pi-snap_checkout">Snap Checkout</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2  ...
```
Sorry can you remove the `style="enable-background:new 0 0 38 24;"` attribute?

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" viewBox="0 0 38 24" aria-labelledby="pi-snap_checkout" xml:space="preserve"><title id="pi-snap_checkout">Snap Checkout</title><path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><g><g><path style="fill:#1A3A67;" d="M7.94,8.66c-0.4,0.32-0.62,0.81-0.61,1.32c-0.02,0.38,0.14,0.75,0.43,1.01 c0.43,0.34,0.95,0.56,1.49,0.66c0.19,0.05,0.38,0.09,0.58,0.12c0.58,0.12,0.89,0.31,0.89,0.57c0,0.17-0.09,0.33-0.25,0.41 c-0.2,0.1-0.43,0.15-0.66,0.14c-0.31,0.03-0.62-0.04-0.89-0.2c-0.2-0.15-0.31-0.39-0.31-0.64h-1.4v0.06 c-0.03,0.54,0.19,1.07,0.61,1.41c0.52,0.36,1.14,0.52,1.77,0.48c0.66,0.05,1.33-0.11,1.9-0.46c0.44-0.34,0.7-0.87,0.67-1.43 c0.01-0.12,0.01-0.23,0-0.35c-0.07-0.32-0.26-0.6-0.53-0.78c-0.27-0.17-0.57-0.3-0.89-0.38c-0.38-0.12-0.78-0.19-1.16-0.29 c-0.19-0.05-0.38-0.11-0.55-0.2c-0.09-0.05-0.22-0.17-0.22-0.28C8.8,9.67,8.88,9.52,9.01,9.44C9.2,9.34,9.42,9.3,9.64,9.31 c0.24-0.01,0.49,0.05,0.69,0.19l0.17,0.17c0.12,0.23,0.36,0.37,0.61,0.36h0.89c-0.02-0.51-0.27-0.99-0.66-1.32 c-0.52-0.35-1.14-0.52-1.77-0.48C8.97,8.19,8.38,8.36,7.91,8.7"/><path style="fill:#1A3A67;" d="M7.94,8.66c-0.4,0.32-0.62,0.81-0.61,1.32c-0.02,0.38,0.14,0.75,0.43,1.01 c0.43,0.34,0.95,0.56,1.49,0.66c0.19,0.05,0.38,0.09,0.58,0.12c0.58,0.12,0.89,0.31,0.89,0.57c0,0.17-0.09,0.33-0.25,0.41 c-0.2,0.1-0.43,0.15-0.66,0.14c-0.31,0.03-0.62-0.04-0.89-0.2c-0.2-0.15-0.31-0.39-0.31-0.64h-1.4v0.06 c-0.03,0.54,0.19,1.07,0.61,1.41c0.52,0.36,1.14,0.52,1.77,0.48c0.66,0.05,1.33-0.11,1.9-0.46c0.44-0.34,0.7-0.87,0.67-1.43 c0.01-0.12,0.01-0.23,0-0.35c-0.07-0.32-0.26-0.6-0.53-0.78c-0.27-0.17-0.57-0.3-0.89-0.38c-0.38-0.12-0.78-0.19-1.16-0.29 c-0.19-0.05-0.38-0.11-0.55-0.2c-0.09-0.05-0.22-0.17-0.22-0.28C8.8,9.67,8.88,9.52,9.01,9.44C9.2,9.34,9.42,9.3,9.64,9.31 c0.24-0.01,0.49,0.05,0.69,0.19l0.17,0.17c0.12,0.23,0.36,0.37,0.61,0.36h0.89c-0.02-0.51-0.27-0.99-0.66-1.32 c-0.52-0.35-1.14-0.52-1.77-0.48C8.97,8.19,8.38,8.36,7.91,8.7"/></g><g> <path style="fill:#1A3A67;" d="M12.45,10.59L12.45,10.59v3.31h1.47v-3.36c0-0.54,0.44-0.97,0.97-0.98 c0.54,0,0.97,0.44,0.98,0.97c0,0,0,0,0,0v2.66c0,0.19,0.07,0.38,0.19,0.52c0.14,0.13,0.33,0.2,0.52,0.19h0.73v-3.35 c0-1.35-1.09-2.44-2.44-2.44s-2.44,1.09-2.44,2.44"/><path style="fill:#1A3A67;" d="M12.45,10.59L12.45,10.59v3.31h1.47v-3.36c0-0.54,0.44-0.97,0.97-0.98 c0.54,0,0.97,0.44,0.98,0.97c0,0,0,0,0,0v2.66c0,0.19,0.07,0.38,0.19,0.52c0.14,0.13,0.33,0.2,0.52,0.19h0.73v-3.35 c0-1.35-1.09-2.44-2.44-2.44s-2.44,1.09-2.44,2.44"/></g><g><path style="fill:#1A3A67;" d="M19.14,12.74c-0.26-0.21-0.3-0.59-0.09-0.85c0.01-0.01,0.02-0.03,0.03-0.04 c0.23-0.27,0.73-0.29,1.08-0.36c0.3-0.05,0.6-0.13,0.89-0.25v0.48c0.02,0.32-0.11,0.63-0.35,0.83c-0.27,0.21-0.61,0.33-0.96,0.3 C19.59,12.85,19.14,12.74,19.14,12.74z M18.43,8.61c-0.34,0.26-0.55,0.65-0.59,1.07h1.4c0.06-0.15,0.19-0.27,0.35-0.32 c0.38-0.13,1.45-0.2,1.45,0.43c0,0.48-0.8,0.53-1.13,0.58c-0.6,0.04-1.19,0.23-1.7,0.56c-0.41,0.31-0.62,0.82-0.55,1.33 c-0.05,0.52,0.16,1.03,0.57,1.36c0.38,0.28,0.85,0.42,1.32,0.42c0.3,0,0.59-0.02,0.89-0.05c0.25-0.03,0.5-0.09,0.73-0.19l0,0 c0.12,0.14,0.3,0.23,0.49,0.24h0.72V9.75c0.02-0.47-0.22-0.92-0.61-1.18c-0.54-0.3-1.16-0.44-1.77-0.4 c-0.58-0.04-1.15,0.13-1.61,0.49"/></g><g><path style="fill:#1A3A67;" d="M29.81,7.04c-0.4,0-0.73,0.33-0.73,0.73l0.4,4.23h1.07l0.24-4.95L29.81,7.04z"/><path style="fill:#1A3A67;" d="M29.81,7.04c-0.4,0-0.73,0.33-0.73,0.73l0.4,4.23h1.07l0.24-4.95L29.81,7.04z"/></g><g><path style="fill:#1A3A67;" d="M29.37,12.57v1.36h0.8c0.25,0.03,0.48-0.15,0.51-0.4c0-0.04,0-0.08,0-0.11v-0.89L29.37,12.57 z"/><path style="fill:#1A3A67;" d="M29.37,12.57v1.36h0.8c0.25,0.03,0.48-0.15,0.51-0.4c0-0.04,0-0.08,0-0.11v-0.89 L29.37,12.57z"/></g><path style="fill:#98C354;" d="M7.64,16.97v-1.38H7.4v-0.29h0.24v-0.27c0-0.27,0.17-0.42,0.5-0.42H8.4v0.35H8.28c-0.15,0-0.23,0.06-0.23,0.21 v0.12h0.36v0.28H8.05v1.39H7.64z"/><path style="fill:#98C354;" d="M9.01,14.61h0.41v0.42H9.01V14.61z M9.01,15.29h0.41v1.68H9.01V15.29z"/><path style="fill:#98C354;" d="M10.19,16.97v-1.67h0.41v0.17c0.1-0.13,0.25-0.2,0.42-0.2c0.28-0.04,0.53,0.15,0.58,0.42 c0.01,0.05,0,1.3,0,1.3h-0.36v-1.02c0-0.2-0.1-0.3-0.28-0.3c-0.09,0-0.18,0.03-0.25,0.1c-0.08,0.1-0.11,0.22-0.11,0.35v0.89h-0.4 V16.97z"/><path style="fill:#98C354;" d="M13.82,16.69c-0.01,0.1,0.01,0.19,0.05,0.28h-0.42c-0.03-0.04-0.05-0.1-0.04-0.15 c-0.15,0.12-0.34,0.19-0.54,0.19c-0.15,0.01-0.29-0.04-0.41-0.14c-0.11-0.1-0.17-0.24-0.16-0.38c0-0.4,0.67-0.52,0.67-0.52 l0.43-0.16l0,0c0-0.14-0.1-0.2-0.32-0.2c-0.08-0.01-0.16,0.01-0.23,0.05c-0.05,0-0.07,0.1-0.07,0.19h-0.4 c-0.01-0.17,0.07-0.33,0.21-0.43c0.15-0.09,0.32-0.14,0.49-0.13c0.17-0.01,0.34,0.02,0.5,0.1c0.16,0.09,0.25,0.27,0.23,0.45 M13.41,16.4v-0.24c-0.06,0.03-0.12,0.05-0.19,0.06L13,16.28l-0.19,0.06c-0.05,0.04-0.09,0.1-0.08,0.17c0,0.06,0.03,0.11,0.07,0.14 c0.05,0.04,0.12,0.06,0.19,0.06c0.1,0,0.2-0.02,0.28-0.07C13.35,16.59,13.41,16.5,13.41,16.4"/><path style="fill:#98C354;" d="M14.56,16.97v-1.67h0.41v0.17c0.11-0.13,0.28-0.21,0.45-0.2c0.28-0.04,0.54,0.16,0.58,0.44 c0.01,0.04,0.01,0.09,0,0.13v1.14h-0.4v-1.02c0-0.2-0.09-0.3-0.28-0.3c-0.09,0-0.18,0.03-0.25,0.1c-0.07,0.1-0.11,0.22-0.1,0.35 v0.89h-0.41V16.97z"/><path style="fill:#98C354;" d="M18.25,15.73l-0.35,0.17c-0.03-0.08-0.07-0.15-0.13-0.2c-0.06-0.05-0.14-0.07-0.22-0.07 c-0.13,0-0.25,0.07-0.32,0.18c-0.08,0.1-0.11,0.23-0.11,0.35c0,0.12,0.03,0.24,0.1,0.35c0.07,0.1,0.19,0.16,0.31,0.16 c0.19-0.01,0.36-0.14,0.41-0.33l0.34,0.19c-0.11,0.32-0.42,0.52-0.75,0.51c-0.23,0.01-0.45-0.08-0.6-0.25 c-0.15-0.17-0.23-0.4-0.22-0.63c-0.01-0.23,0.07-0.46,0.23-0.64c0.17-0.17,0.39-0.25,0.63-0.24c0.31-0.02,0.6,0.17,0.7,0.47"/><path style="fill:#98C354;" d="M20.44,16.23h-1.23c0,0.12,0.05,0.23,0.14,0.3c0.08,0.07,0.18,0.11,0.29,0.12c0.19,0.01,0.37-0.08,0.48-0.24 l0.34,0.19c-0.17,0.28-0.41,0.42-0.82,0.42c-0.27,0-0.42-0.07-0.63-0.26c-0.15-0.17-0.23-0.4-0.22-0.63 c-0.01-0.23,0.07-0.46,0.23-0.63c0.17-0.16,0.39-0.25,0.62-0.25C20.19,15.28,20.39,15.59,20.44,16.23 M19.21,15.92h0.82 c-0.01-0.07-0.04-0.14-0.09-0.19c-0.07-0.09-0.18-0.14-0.29-0.14c-0.11,0-0.21,0.03-0.29,0.1C19.28,15.75,19.23,15.83,19.21,15.92"/><path style="fill:#1A3A67;" d="M25.68,8.17c-1.58,0-2.85,1.28-2.85,2.86l0,0v4.92h0.71c0.19,0,0.37-0.08,0.51-0.21 c0.14-0.13,0.21-0.32,0.2-0.51v-1.77c0.43,0.26,0.92,0.4,1.43,0.4c1.58,0,2.86-1.28,2.86-2.86c0-1.58-1.28-2.86-2.86-2.86 L25.68,8.17z M25.73,12.49c-0.81,0-1.47-0.65-1.47-1.45c0-0.81,0.65-1.47,1.45-1.47c0.81,0,1.47,0.65,1.47,1.45 c0,0.01,0,0.01,0,0.02C27.18,11.84,26.53,12.49,25.73,12.49z"/></g></svg>
```

spatel-snap replied:
Removed the unused attributes, can you please review it agian?

tauthomas01 reviewed (changes requested):
Just found one last thing and I'm approving it.

larouxn reviewed (approved):
Looks good to me, assuming we want this colour scheme. :+1: 
![Screenshot from 2022-03-04 18-56-44](https://user-images.githubusercontent.com/1557529/156741845-9779646d-158e-48b3-bd78-fe6ef3023bde.png)

tauthomas01 reviewed (approved):
Looks good now

randallalv commented:
Hello Shopify team, how much time does this take to be available on our Shopify partner account to update our payment app icon? @larouxn @tauthomas01

larouxn commented:
> Hello Shopify team, how much time does this take to be available on our Shopify partner account to update our payment app icon?

:wave: We typically batch icon releases but thankfully already have some in the pipeline so we're pretty much ready for a release. Once this is merged, I would expect it to be live within a week or two.

larouxn commented:
👋 Hey @randallalv, we're still in the process of getting the icon live. As mentioned in https://github.com/activemerchant/payment_icons/pull/567#issuecomment-1059885409 we typically batch icons, make a gem release, then rollout to everywhere in Shopify. At this time, I estimate this new icon will be available everywhere next week.

larouxn commented:
👋 Update on our end, we're planning to have this new icon live this week. Will update here when it's available.

larouxn commented:
👋 Hello there, your new icon should be live everywhere now.

spatel-snap commented:
![image](https://user-images.githubusercontent.com/65699837/158763783-90600e9a-8f81-4db3-bb7d-d3d6aeac462c.png)
Please find the gift icon displayed on the setting page.

randallalv commented:
Hello team! I just submitted the new app extension with the new Snap Finance icon.
Please approve the version 4.6.0.
Thanks!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: spatel-snap ***@***.***>
Sent: Thursday, March 17, 2022 2:03:15 AM
To: activemerchant/payment_icons ***@***.***>
Cc: Randall Josue Alvarado Barboza ***@***.***>; Mention ***@***.***>
Subject: Re: [activemerchant/payment_icons] Snap Checkout new icon (PR #567)


[image]<https://isolate.menlosecurity.com/1/3735926911/https://user-images.githubusercontent.com/65699837/158763783-90600e9a-8f81-4db3-bb7d-d3d6aeac462c.png>
Please find the gift icon displayed on the setting page.

—
Reply to this email directly, view it on GitHub<https://github.com/activemerchant/payment_icons/pull/567#issuecomment-1070510471>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ARMDCGLPB2FFDUHI4YVT5R3VALRMHANCNFSM5PGPQSTA>.
Triage notifications on the go with GitHub Mobile for iOS<https://isolate.menlosecurity.com/1/3735926911/https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://isolate.menlosecurity.com/1/3735926911/https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
